test/buy-collateral-test.ts

```diff
@@ -321,12 +393,12 @@ describe('buyCollateral', function () {
 
       // set collateral balance
       const t0 = Object.assign({}, await normalComet.totalsCollateral(normalWETH.address), {
-        totalSupplyAsset: exp(100, 18),
+        totalSupplyAsset: exp(5, 18),
```

Comment:
Curious what these changes are for?

Reply:
These are necessary bc you set up collateral reserves now differently than before. This is the amount of collateral that belongs to users excluding reserves now

Reply:
In that case, shouldn't these (`totalSupplyAsset`) stay at 100e18 while lines 421-430 change from `setCollateralBalance of Comet for WETH to 5e18` to `subtract 5e18 from totalSupplyAsset for WETH`?

Reply:
Hm, that's a good point, I guess the `setCollateralBalance` is supposed to be simulating absorb here, not a `supplyCollateral`... but I guess I don't really understand this part of the test. I can just delete the setting of totals collateral and collateral balance for the market, it doesn't change any of whats being tested, afaik